pyproject.toml

```diff
@@ -218,3 +218,22 @@ showcontent = true
 directory = "test"
 name = "Test"
 showcontent = true
+
+[tool.bandit]
+exclude_dirs = ["tests"]
+skips = [
+    "B105",
+    "B110",
+    "B112",
+    "B307",
+    "B310",
+    "B311",
+    "B314",
+    "B403",
+    "B404",
+    "B405",
+    "B602",
+    "B603",
+    "B604",
+    "B607",
+    ]
```

Comment:
This makes whole source code `# nosec` free. We can expand it in future if needed.